samples/Patient and Population Services G10/infra/main.bicep

```diff
@@ -9,63 +11,265 @@ param name string
 @description('Primary location for all resources')
 param location string
 
-@description('Id of the user or app to assign application roles')
+@description('Id of the user or app to assign application roles. Passed in by Azure Developer CLI.')
 param principalId string = ''
 
-@description('Audience for SMART scopes in Azure Active Directory. Leave blank to use the PaaS Service URL.')
-param smartAudience string = ''
+// end azd populated parameters
+
+// start required API gateway parameters
 
 @description('Name of the owner of the API Management resource')
-param apimPublisherName string
+param apimPublisherName string = 'Test Name'
 
 @description('Email of the owner of the API Management resource')
-param apimPublisherEmail string
+param apimPublisherEmail string = 'test@test.com'
 
 @description('ClientId for the context static app registration for this FHIR Service (you must create this)')
-param contextAadApplicationId string
+param contextAadApplicationId string = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
```

Comment:
Static app registration id is hard-coded. Was this intended?

Reply:
Thanks, had some other changes sneak in.